### PR TITLE
migrate windows CI from appveyor to GA

### PR DESCRIPTION
**Pull request by orklah**
This PR propose ditching Appveyor in favor of Github Actions.

I did a test without cache and GA seem to be slightly worse in execution time so I hope with cache they'll be equivalent.

The big issue we have with Appveyor is that the queue is shared between all PR and branches in the repo. This means when there is a lot of activity, Appveyor queue can get really long. I don't thing GA has this issue.

I'm not really good with yaml files nor with CI builds so it's probable I made mistakes here.

### CODE REVIEW

**Comment by orklah** [comment]
Appveyor launched itself here though I deleted the file in the PR. Not sure if it uses the one in master or if I forgot to delete somethin

**Comment by weirdan** [comment]
> Appveyor launched itself here though I deleted the file in the PR.

I think @muglug needs to disable it in repo settings.

**Comment by weirdan** [comment]
> simplify windows ci 

Where there some problems with running tests in chunks?

**Comment by orklah** [comment]
> Where there some problems with running tests in chunks?

Yeah, It uses this file: https://github.com/vimeo/psalm/blob/master/bin/tests-github-actions.sh
And I had no idea how to translate it for windows

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+build: false
+branches:
+  only:
+    - dear-appveyor-please-dont-build-this-repo
```
Comment (orklah) [inline .appveyor.yml:4]: 😄

Comment (weirdan) [reply .appveyor.yml:4]: `¯\_(ツ)_/¯`

**Comment by weirdan** [comment]
Thanks!

**Pull request merged**